src/OrchardCore.Modules/OrchardCore.Media/Views/Items/MediaDeploymentStep.Fields.Edit.cshtml

```diff
@@ -8,15 +8,23 @@
             <div class="form-check">
                 <label class="form-check-label">
                     <input class="form-check-input" asp-for="IncludeAll"
-                           data-reverseToggle=".mediasel" />
+                           data-reverseToggle=".media-sel" />
                     @T["Include all media."]
                 </label>
             </div>
         </div>
     </div>
 </fieldset>
 
-<fieldset class="mediasel form-group mt-4 mb-5 @(Model.IncludeAll ? "collapse" : "show")">
+<h6>NOTE:</h6>
+<ul>
+    <li>Selecting "Include all media." will ensure that all media is added to the package when this deployment plan executes, regardless of what you see here now.</li>
```

Comment:
Too much information, should go in the docs. And it's not translated. Better keep it simple such that translations are easier to build. And details can go in the documentation.